src/components/other/Links.js

```diff
@@ -52,6 +53,41 @@ export default class Links extends Component {
             </tr>
           </tbody>
         </table>
+        <br />
+        <table className="table table-striped table-bordered table-hover">
+          <tbody>
+            <tr className="info">
+              <td className="tableHeader">{this.context.t('links.sponsors')}</td>
+            </tr>
+            <tr>
+              <td><a href="http://vdhkeukens.be/" target="_blank">{this.context.t('links.keuken')}</a></td>
```

Comment:
sponsors moeten niet in locales want die gaan te moeilijk zijn om te tonen
elke is anders, met foto enzo... deze gaan hardcoded :)

ik zou ze ook bovenaan zetten
of mss rond de rest, zodat je de links nog goed kan zien maar ondertussen de sponsers ook zichtbaar zijn?